scripts/smg-html.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
+
+import os
+import re
+import json
+
+# create directories if don't exist
+dirMain = "../output/smg-html"
+dirImg = "../output/smg-html/img"
+dirOutput = "../output"
```

Comment:
The base path (../output) should be configurable (passed as a parameter) rather than hardcoded. The script should not depend on call location.